noxfile.py

```diff
@@ -39,7 +39,7 @@ def black(session):
 def lint(session):
     args = session.posargs or locations
     session.install("flake8", "flake8-bandit", "flake8-black", "flake8-bugbear")
-    session.run("flake8", *args)
+    session.run("flake8", "--exclude=wetterdienst/cli.py", *args)
```

Comment:
Please move this to the .flake8 config. I have already added two files that are ignored by flake8.

Reply:
Btw you can also precisely choose the errors that should be ignore (probably line length in this case)

Reply:
Thanks. I've adjusted the PR accordingly.